src/Microsoft.AspNet.Mvc.Core/ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -179,7 +179,12 @@ public override string Description
         {
             get
             {
-                return DisplayMetadata.Description;
+                if (DisplayMetadata.Description == null)
+                {
+                    return null;
+                }
+
+                return DisplayMetadata.Description();
```

Comment:
you are right .. dont know why i overlooked while writing it .. thanks for pointing out.